docs/markdown/authoritative/backend-generic-mysql.md

```diff
@@ -0,0 +1,87 @@
+# Generic MySQL backend
```

Comment:
maybe just drop the `Generic` now that there are separate docs for each backend?